client/signup/steps/design-type-with-store/bluehost-store/index.jsx

```diff
@@ -0,0 +1,66 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Gridicon from 'components/gridicon';
+import FormButton from 'components/forms/form-button';
+import StepHeader from 'signup/step-header';
+import Button from 'components/button';
+import { abtest } from 'lib/abtest';
+import { localize } from 'i18n-calypso';
+
+function redirectToPartner() {
+	if ( 'bluehostWithWoo' === abtest( 'signupStoreBenchmarking' ) ) {
+		window.location.href = 'https://www.bluehost.com/web-hosting/signup?flow=woocommerce';
+	} else {
+		window.location.href = 'https://www.bluehost.com/wordpress';
+	}
+}
+
+function getPrice() {
+	return 'bluehost' === abtest( 'signupStoreBenchmarking' ) ? '$3.95' : '$12.95';
+}
+
+export const BluehostStoreStep = props => {
```

Comment:
Only have two props? We could inline them…

```js
export const BluehostStoreStep = ( { onBackClick, translate } ) => (
	<div>
		…
	</div>
);
```

Some will disagree because this could lead to bigger diffs if we extend the component. I'm comfortable with this for simple static components like this.